reviews/views.py

```diff
@@ -73,101 +79,136 @@ class DecisionOpenView(BaseDecisionListView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
-        context['title'] = _("Openstaande besluiten commissieleden")
-        context['list_template'] = "reviews/vue_templates/decision_list_reviewer.html"
-        context['data_url'] = reverse("reviews:api:open", args=[self.committee])
+        context["title"] = _("Openstaande besluiten commissieleden")
+        context["list_template"] = "reviews/vue_templates/decision_list_reviewer.html"
+        context["data_url"] = reverse("reviews:api:open", args=[self.committee])
 
         return context
-    
 
-class CommitteeMembersWorkloadView(GroupRequiredMixin, CommitteeMixin, generic.TemplateView):
-    
-    template_name = 'reviews/committee_members_workload.html'
+
+class CommitteeMembersWorkloadView(
+    GroupRequiredMixin, CommitteeMixin, generic.FormView
+):
+    template_name = "reviews/committee_members_workload.html"
     group_required = [settings.GROUP_SECRETARY]
-    
+    form_class = StartEndDateForm
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.today = date.today()
+        self.one_year_ago = self.today - timedelta(days=365)
+
+    def get_initial(self):
+        initial = super().get_initial()
+
+        if "start_date" in self.request.GET and "end_date" in self.request.GET:
+            initial["start_date"] = self.request.GET.get("start_date")
+            initial["end_date"] = self.request.GET.get("end_date")
+        else:
+            initial["start_date"] = self.one_year_ago.strftime("%Y-%m-%d")
+            initial["end_date"] = self.today.strftime("%Y-%m-%d")
+
+        return initial
```

Comment:
The way you're using the form here doesn't really make sense. It's basically a way to only display the form, which would probably be better done with just plain HTML at that point. 

It works... But you're leaving a lot functionality on the table, most importantly validation.

I would suggest actually using the form-processing; This is done by making the form use 'post', and then overriding the 'post()' method of this View to retrieve the validated and cleaned data. For example:
```python

    def __init__(self, **kwargs):
        super().__init__(**kwargs)
        self.today = date.today()
        self.start_date = self.today - timedelta(days=365)
        self.end_date = self.today

    def post(self, request, *args, **kwargs):
        form = self.get_form()
        if form.is_valid():
            self.start_date = form.cleaned_data['start_date']
            self.end_date = form.cleaned_data['end_date']
        else:
            return self.form_invalid(form)

        return self.get(request, *args, **kwargs)

    def get_initial(self):
        initial = super().get_initial()

        # You don't actually have to cast these to strings btw, DateField can handle date(time)s
        initial["start_date"] = self.start_date
        initial["end_date"] = self.end_date

        return initial
```

Then, use `self.start_date` and `self.end_date` in your querysets